<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="krish" failures="2" tests="2" name="Test" time="0.857" errors="0" timestamp="2023-03-27T12:09:07 IST">
  <testcase classname="com.HRM_copy.Admin_login01" name="AdminLogin" time="0.738">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at com.HRM_copy.Admin_login01.AdminLogin(Admin_login01.java:22)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
at java.base/java.lang.Thread.run(Thread.java:834)
... Removed 18 stack frames]]>
    </failure>
  </testcase> <!-- AdminLogin -->
  <testcase classname="com.HRM_copy.Create_Branch02" name="create_Branch02" time="0.068">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at com.HRM_copy.Create_Branch02.create_Branch02(Create_Branch02.java:23)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
at java.base/java.lang.Thread.run(Thread.java:834)
... Removed 18 stack frames]]>
    </failure>
  </testcase> <!-- create_Branch02 -->
</testsuite> <!-- Test -->
